src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/DfaMatchingState.cs

```diff
@@ -30,19 +32,19 @@ internal DfaMatchingState(SymbolicRegexNode<T> node, uint prevCharKind)
         internal bool IsDeadend => Node.IsNothing;
 
         /// <summary>The node must be nullable here</summary>
-        internal int WatchDog
+        internal int FixedLength
```

Comment:
NIT: Adjust triple slash docs as well.

Reply:
The renaming of `WatchDog` will cause merge conflict with my recent PR, but that should be fine. I may ask help with merging ... or someone else may merge :-)